### PR TITLE
Demos: improve accessibility

### PR DESCRIPTION
**Pull request by AlexanderMoiseev**

### CODE REVIEW

File: JSDemos/Demos/Calendar/Overview/Angular/app/app.component.html [hunk 1]
```diff
@@ -70,6 +70,7 @@
       <span>First day of week</span>
       <dx-select-box
         [dataSource]="weekDays"
+        [inputAttr]="{ 'aria-label': 'Day' }"
```
Comment (ksercs) [inline JSDemos/Demos/Calendar/Overview/Angular/app/app.component.html]:
```suggestion
        [inputAttr]="{ 'aria-label': 'First Day of Week' }"
```
Please, update is all approcahes

File: JSDemos/Demos/Calendar/Overview/Angular/app/app.component.html [hunk 2]
```diff
@@ -79,12 +80,17 @@
       <span>Week number rule</span>
       <dx-select-box
         [dataSource]="weekNumberRules"
+        [inputAttr]="{ 'aria-label': 'Rule' }"
```
Comment (ksercs) [inline JSDemos/Demos/Calendar/Overview/Angular/app/app.component.html]:
```suggestion
        [inputAttr]="{ 'aria-label': 'Week Number Rule' }"
```
plz, update in all approaches

File: JSDemos/Demos/Resizable/Overview/Angular/app/app.component.html [hunk 3]
```diff
@@ -47,7 +47,7 @@
     <dx-tag-box
       [items]="handleValues"
       [(value)]="handles"
-      [inputAttr]="{ 'aria-label': 'Handle' }"
+      [inputAttr]="{ 'aria-label': 'Behavior' }"
```
Comment (ksercs) [inline JSDemos/Demos/Resizable/Overview/Angular/app/app.component.html]: y did u change it?
```suggestion
      [inputAttr]="{ 'aria-label': 'Handle' }"
```

File: JSDemos/Demos/RangeSelector/UseRangeSelectionForCalculation/React/App.js [hunk 4]
```diff
@@ -8,6 +8,8 @@ const startValue = new Date(2011, 0, 1);
 const endValue = new Date(2011, 11, 31);
 const behaviorModes = ['onHandleMove', 'onHandleRelease'];
 
+const behaviorLabel = { 'aria-label': 'Behavior' };
```
Comment (ksercs) [inline JSDemos/Demos/RangeSelector/UseRangeSelectionForCalculation/React/App.js]:
```suggestion
const valueChangeModeLabel = { 'aria-label': 'Value Change Mode' };
```
Please, update also the appropriate demo code and fix it in all frameworks

File: JSDemos/Demos/SelectBox/Overview/Angular/app/app.component.html [hunk 5]
```diff
@@ -40,6 +46,7 @@
     <div class="dx-field-value">
       <dx-select-box
         [dataSource]="data"
+        [inputAttr]="{ 'aria-label': 'Product From DS' }"
```
Comment (ksercs) [inline JSDemos/Demos/SelectBox/Overview/Angular/app/app.component.html]:
```suggestion
        [inputAttr]="{ 'aria-label': 'Product ID' }"
```
plz, update in all frameworks

File: JSDemos/Demos/TreeList/FilterModes/React/data.js [hunk 6]
```diff
@@ -390,3 +390,4 @@ export const employees = [{
   Hire_Date: '2009-04-22',
 }];
 
+export const filterLabel = { 'aria-label': 'Filter' };
```
Comment (ksercs) [inline JSDemos/Demos/TreeList/FilterModes/React/data.js:393]: I don't see a place where u used it

File: JSDemos/Demos/TreeList/MultipleRowSelection/React/data.js [hunk 7]
```diff
@@ -390,3 +390,4 @@ export const employees = [{
   Hire_Date: '2009-04-22',
 }];
 
+export const selectionModeLabel = { 'aria-label': 'Selection Mode' };
```
Comment (ksercs) [inline JSDemos/Demos/TreeList/MultipleRowSelection/React/data.js:393]: I don't see a place where u used it

File: JSDemos/Demos/TreeList/Resizing/React/data.js [hunk 8]
```diff
@@ -390,3 +390,4 @@ export const employees = [{
   Hire_Date: '2009-04-22',
 }];
 
+export const columnResizingModeLabel = { 'aria-label': 'Column Resizing Mode' };
```
Comment (ksercs) [inline JSDemos/Demos/TreeList/Resizing/React/data.js:393]: I don't see a place where u used it

File: JSDemos/Demos/TreeList/FilterModes/React/data.js (hunk 6 above)
Comment (AlexanderMoiseev) [reply JSDemos/Demos/TreeList/FilterModes/React/data.js:393]: thanks, missed that. seems like happened with saveAll in vs code when there were a lot of changes, coz i see unsaved files now

File: JSDemos/Demos/Resizable/Overview/Angular/app/app.component.html (hunk 3 above)
Comment (AlexanderMoiseev) [reply JSDemos/Demos/Resizable/Overview/Angular/app/app.component.html]: opps )

**Review by ksercs — APPROVED** [review]

**Review by marker-dao — APPROVED** [review]

**Pull request merged**